projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/use-protect-tooltip-copy.ts

```diff
@@ -82,9 +201,30 @@ export function useProtectTooltipCopy( {
 							'jetpack-my-jetpack'
 						),
 				  },
-		autoFirewallTooltip: {
-			title: __( 'Auto-Firewall: Inactive', 'jetpack-my-jetpack' ),
-			text: __( 'Upgrade required for activation. Manual rules available.', 'jetpack-my-jetpack' ),
-		},
+		autoFirewallTooltip:
+			hasProtectPaidPlan && ! isAutoFirewallEnabled
+				? {
+						title: __( 'Auto-Firewall: Inactive', 'jetpack-my-jetpack' ),
+						text: createInterpolateElement(
+							__(
+								'You have Auto-Firewall disabled, visit your Protect <a>firewall settings</a> to activate.',
+								'jetpack-my-jetpack'
+							),
+							{
+								a: createElement( 'a', {
+									href: 'admin.php?page=jetpack-protect#/firewall',
```

Comment:
Is there any reason we can't use the `settingsLink` variable here as well? If we are sending them to a protect admin page I assume that Jetpack Protect is installed, and if that is the case, settingsLink should be the same value as we have hardcoded here

Reply:
Ahh yes, thanks.  I overlooked this one.